src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage_validators.py

```diff
@@ -116,5 +116,59 @@ def test_storage_get_char_options_validator(self):
         self.assertEqual(result, set('ab'))
 
 
+@api_version_constraint(resource_type=ResourceType.MGMT_STORAGE, min_api='2016-12-01')
```

Comment:
Since part of this PR is a fix for the older API versions #4220, we should also include a test for that API version to ensure that the fix in fact fixes that issue.